test/unit/test_list.c

```diff
@@ -218,6 +219,30 @@ bool test_rz_list_sort5(void) {
 	mu_end;
 }
 
+bool test_rz_list_from_iter(void) {
+	HtUP *alpha_ht = ht_up_new(NULL, NULL);
+	char *unordered_alphabeth[] = { "b", "w", "k", "a", "c", "d", "e", "f", "g", "h", "i", "j", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "x", "y", "z" };
+	char *lower[] = { "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z" };
+
+	for (size_t i = 0; i < 26; i++) {
+		ht_up_insert(alpha_ht, i, (void *)unordered_alphabeth[i]);
+	}
+	RzIterator *iter = ht_up_as_iter(alpha_ht);
+	RzList *list = rz_list_new_from_iterator(iter);
```

Comment:
`mu_assert_neq(list, NULL)`

Reply:
there is a function for checking for null.